Makefile

```diff
@@ -0,0 +1,19 @@
+ROOT_DIR:=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
+
+ifndef CHANNEL
+$(error CHANNEL is not set. Must set CHANNEL=value in the build command.)
+endif
+ifndef VERSION
+$(error VERSION is not set. Must set VERSION=value in the build command.)
+endif
+
+.PHONY: build
+build: builder
+	docker pull archlinux:base-devel
+	docker run --rm -v $(ROOT_DIR)/output:/output --privileged=true image-builder:latest $(CHANNEL) $(VERSION) 
```

Comment:
The command needs to include the git commit hash instead of CHANNEL and VERSION

This is actually my local makefile:
```
system_image: build_image
        docker run --rm -v $(pwd)/output:/output --privileged=true image-builder:latest $(git rev-parse HEAD | cut -c1-7)

build_image:
        docker build -t image-builder:latest .
```